src/solver/SolverController.cpp

```diff
@@ -563,31 +565,35 @@ void SolverController::temperature_source() {
 /// \brief  Updates time dependent parameters force source functions
 // ***************************************************************************************
 void SolverController::force_source() {
-    auto params = Parameters::getInstance();
     // Force
-    if (params->get("solver/source/force_fct") == SourceMethods::Buoyancy) {
-        std::string dir = params->get("solver/source/dir");
-        if (params->get("solver/source/use_init_values") == XML_FALSE) {
-            real ambient_temperature_value = params->get_real("solver/source/ambient_temperature_value");
-            //m_field_controller->get_field_T().set_value(ambient_temperature_value);
-            m_field_controller->get_field_T_ambient().set_value(ambient_temperature_value);
-        }
+    if (m_settings.get("solver/source/force_fct") != SourceMethods::Buoyancy) {
+        return;
+    }
```

Comment:
the inquiry if the source method buoyancy is used, is intentional (for future use, when more source methods are implemented )

Reply:
done in 247b6d2